### PR TITLE
resolve issue editProfile, resolve SignIn styles, created UserProfile…

### PR DESCRIPTION
**Pull request by IgorPopach**
… component template

### CODE REVIEW

File: client/src/components/AppHeader/AppHeader.js [hunk 1]
```diff
@@ -45,13 +44,11 @@ class AppHeader extends React.Component {
...
 
     render() {
-        console.log('appheader props', this.props)
         const { user, isAuthenticated } = this.props;
         const { mobileMenuOpened, activePage } = this.state;
         const avatarURL = user && user.avatar ? 
             <img src={user.avatar} alt={user.email}/> 
-            : <img src={face} alt ="user" />
-        console.log('[appheader user]', user)
+            : <img src={face} alt ="user" />;
```
Comment (yodeco) [inline client/src/components/AppHeader/AppHeader.js]: Better add user name or just leave empty alt attribute. Not critical, but if possible, enhance that moment pls.

File: client/src/components/SignIn/SignIn.scss [hunk 2]
```diff
@@ -23,7 +22,7 @@
 
 // Left part;
 .sign-in .sign-in__info {
-    display: none;
+    display: none !important;
     width: 100%;
```
Comment (yodeco) [inline client/src/components/SignIn/SignIn.scss:26]: !important, remove it by making more specific selector, or add comment with explanations

**Review by yodeco — APPROVED** [review]

File: client/src/components/AppHeader/AppHeader.js (hunk 1 above)
Comment (IgorPopach) [reply client/src/components/AppHeader/AppHeader.js]: Ok! I'll do it in the next pull request

File: client/src/components/SignIn/SignIn.scss (hunk 2 above)
Comment (IgorPopach) [reply client/src/components/SignIn/SignIn.scss:26]: I tryed but they couldn't help me...

**Review by Blager89 — APPROVED** [review]

**Pull request merged**